src/server/server.ts

```diff
@@ -35,6 +35,7 @@ namespace ts.server {
     } = require("os");
 
     function getGlobalTypingsCacheLocation() {

```

Comment:
So is each different version location going to have its own types registry and package.json? Seems like it. We should make sure this location is consistently used across installing and using these.